### PR TITLE
added a new section for copyDependencies

### PR DESCRIPTION
**Pull request by saumyapandyaa**
issue: https://github.com/OpenLiberty/guide-maven-intro/issues/34

### CODE REVIEW

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]
hi @saumyapandyaa 

I suggest to move the "Third-party libraries for Open Liberty with Maven" subsection under "Configuring JPA" section.
- it should similar to the content in deepdive guide 
   - "To use a PostgreSQL database, ..." -> "To use a Derby database, ..."
   - replace the pom.xml
-  the subsection title to be "Setting database libraries"

**Comment by gkwan-ibm** [comment]
dev content review:
- move the subsection to a new section after the "Performing CRUD operations using JPA" section
- rename the subsection title to "Configuring the Derby driver and the Liberty"
- explain adding 3 derby dependencies in `<dependencies></dependencies>` to download the derby libraries to local
- explain using `copyDependencies` to copy the derby libraries to Liberty shared resources directory
- explain the shared resources directory is used in the `derbyJDBCLib` of server.xml
  - hotspot to this [lines](https://github.com/OpenLiberty/guide-jpa-intro/blob/6c87982f3039b451893daa6e7dce9a45b722ae70/finish/backendServices/src/main/liberty/config/server.xml#L20-L22)
- make sure this arrange does not affect the flow that the app can work at the "Running the application" and "Testing the application"

**Review by shin19991207 — APPROVED** [review]
LGTM

File: README.adoc [hunk 1]
```diff
@@ -299,6 +299,35 @@ EventResource.java
 include::finish/backendServices/src/main/java/io/openliberty/guides/event/resources/EventResource.java[]
 ----
 
+// =================================================================================================
+// Configuring the Derby driver and the Liberty
+// =================================================================================================
+
+== Configuring the Derby driver and the Liberty
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
== Configuring the Derby driver and the Liberty Maven plugin
```

File: README.adoc [hunk 2]
```diff
@@ -299,6 +299,35 @@ EventResource.java
...
+include::finish/backendServices/pom.xml[]
+----
+
+server.xml
+[source, Xml, linenums, role='code_column']
+----
+include::finish/backendServices/src/main/liberty/config/server.xml[]
+----
+
+Three required Derby dependencies are added to the [hotspot=dependencies file=0]`dependencies` configuration for Maven to download the Derby libraries locally. The [hotspot=copyDependencies file=0]`copyDependencies` configuration instructs the Liberty Maven plug-in to copy the Derby libraries to Liberty shared resources directory that is specified through the [hotspot=location file=0]`location` configuration, and is referenced in the [hotspot=shared-dir file=1]`derbyJDBCLib` `library` configration of the Liberty `server.xml` configuration file.
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
This configuration adds three required Derby dependencies to the [hotspot=dependencies file=0]`dependencies` configuration so Maven can download the Derby libraries locally. The [hotspot=copyDependencies file=0]`copyDependencies` configuration instructs the Liberty Maven plug-in to copy the Derby libraries to the Liberty shared resources directory that is specified through the [hotspot=location file=0]`location` configuration, and is referenced in the [hotspot=shared-dir file=1]`derbyJDBCLib` `library` configuration of the `server.xml` file.
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review

**Review by dmuelle — APPROVED** [review]

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**